hedera-node/hedera-app/src/main/java/com/hedera/node/app/records/impl/producers/StreamFileProducerConcurrent.java

```diff
@@ -116,8 +116,9 @@ public void initRunningHash(@NonNull final RunningHashes runningHashes) {
                 throw new IllegalStateException("initRunningHash() can only be called once");
             }
 
-            if (runningHashes.runningHash() == null) {
-                throw new IllegalArgumentException("The initial running hash cannot be null");
+            if (runningHashes.runningHash() == null
```

Comment:
If it should be impossible for `runningHash()` to return `null` I would strongly prefer we delete this check.

Reply:
Do you suggest to do that in this back-port to `0.48`, or in a future fix in `develop`?

Reply:
`develop` is fine with me. 💯

Reply:
I filed https://github.com/hashgraph/hedera-services/issues/12031 to address this in `develop`.